src/common/playcreated/play-created.scss

```diff
@@ -21,7 +21,6 @@
       content: "";
       position: absolute;
       height: 4px;
-      background-color: var(--color-brand-primary);
```

Comment:
@Hat52 , Why we are removing this?

Reply:
@Sachin-chaurasiya this is the change from other pull request that is why I this change appears here.

Reply:
removing this line gives us the desired result that I have described in the issue.

Reply:
@Hat52 , This is a common CSS that may affect the other places.

Reply:
@Sachin-chaurasiya I does't effect any other element. I have tested it.

Reply:
@Hat52 , Still I would suggest solving this issue in some other way.

Reply:
@Sachin-chaurasiya okay I will look into it

Reply:
@Sachin-chaurasiya I have looked at the code and this class name is not a generic class Name it is only associated to those text that have a strong element and I have tested this fix and it does't break css anywhere.

Reply:
Thanks, @Hat52 , for verifying it.